snuba/clusters/cluster.py

```diff
@@ -217,20 +212,21 @@ def get_reader(self) -> Reader[SqlQuery]:
     def get_writer(
         self,
         table_name: str,
-        encoder: Callable[[WriterTableRow], bytes],
         options: ClickhouseWriterOptions,
         chunk_size: Optional[int],
     ) -> BatchWriter[WriterTableRow]:
```

Comment:
This will likely change return types at some point, moving the `BatchWriterEncoderWrapper` to a different level of abstraction but I'm not quite sure what/where yet so leaving it here for now.